vtm.js

```diff
@@ -317,7 +467,7 @@ app.component('stat', {
             fill: i > initialValue && i <= stat.value,
             active: hoverPointer && (i === hoverPointer || i > hoverPointer !== i > stat.value)         
           }"      
-          @click="$emit('statChange', [stat, i])"
+          @click="$emit('statChange', [stat, i, ((stat.value >= i) && (i > initialValue))])"
```

Comment:
tomuto nerozumime, akoze niekedy ma to pole tri prvky?

Reply:
ja ti taky teraz nejak nerozumiem

Reply:
no predtym tam bolo:

`[stat, i]` 

to je pole ktore sa skladaz objektu stat a cisla i

a teraz tam bude

`[stat, i, ((stat.value >= i) && (i > initialValue))]`

a to bude pole objekt, i, True alebo False

ale mozno tomu len zle rozumiem

Reply:
tak nwm asi tomu rozumies spravne, ale stale neviem kde je problem

Reply:
tak pozrel som znova a uz tomu rozumiem